charts/prometheus-alerts/templates/containers-prometheusrule.yaml

```diff
@@ -512,4 +512,166 @@ spec:
     {{- end }}
     {{- end }}
 
+  - name: {{ .Release.Name }}.{{ .Release.Namespace }}.alertRulesSelectorsValidity
```

Comment:
Can't these be inside the existing conditions for each group of alerts? Ie, this with the daemonset alerts?